src/posts/lucky_0.27.cr

```diff
@@ -0,0 +1,104 @@
+class Lucky027Release < BasePost
+  title "Lucky v0.27 Brings compatibility with Crystal 1.0"
+  slug "lucky-0_27-release"
+
+  def published_at : Time
+    Time.utc(year: 2021, month: 4, day: 12)
+  end
+
+  def summary : String
+    <<-TEXT
+    Crystal 1.0.0 is out! 🥳 And Lucky
+    0.27.0 is fully compatible with it.
+    TEXT
+  end
+
+  def content : String
+    <<-MD
+    We want to first congratulate the Crystal team for their long awaited release of
+    1.0! This is a huge milestone for not only the language, but the whole community.
+
+    We're sure the next question on your mind is "When will Lucky hit 1.0?". Don't fret!
+    We're working towards that, and with each release we get closer and closer.
```

Comment:
May be worth coming out and saying here what we said on Discord, that it won't be for the next couple releases, at least. It'll be out when it's ready. etc. It's pinned in Discord if you want wording.

Reply:
Cool. I re-worded it slightly. Let me know if it sounds ok. I don't want it to come off as "Stand off-ish" like "don't bug us about it". But instead more like "in reality, this is the true reason.."

Reply:
I like it! I think you give a good reason that doesn't sound like we're annoyed at people for asking.